packages/dapp/src/pages/Invoices.jsx

```diff
@@ -21,6 +22,22 @@ const InvoicesInner = ({ history }) => {
     }
   }, [account, setSearch]);
 
+  useEffect(() => {
+    switch (chainId) {
+      case 1:
+        setNetworkName('Ethereum Mainnet');
+        break;
+      case 4:
+        setNetworkName('Rinkeby');
+        break;
+      case 100:
+        setNetworkName('Gnosis Chain');
+        break;
+      default:
+        setNetworkName(null);
+    }
+  }, [chainId]);
+
```

Comment:
We should probably have a single place with our supported network information. We are going to want to avoid chasing down places where networks are defined when we add support for additional networks.

Reply:
yes, good point. updated by moving network names to the main network info file and referencing from there: [packages/dapp/src/config.js](https://github.com/raid-guild/smart-invoice/pull/94/files#diff-2fb300a5bd2356b12c06754e1433adae1f6d475307903d6fe8ef35d054eb57e1)